nano/node/json_handler.cpp

```diff
@@ -3373,10 +3373,14 @@ void nano::json_handler::receive ()
 					}
 					if (!ec)
 					{
+						// Representative is only used by receive_action when opening accounts
+						// Set a wallet default representative for new accounts
+						nano::account representative (wallet->store.representative (node.wallets.tx_begin_read ()));
```

Comment:
json_handler::receive () has already started a read tx that can be used here right?

Reply:
Yes it does, missed that. Thanks!